pymiele/pymiele.py

```diff
@@ -181,7 +178,7 @@ async def listen_events(
                 )
                 await asyncio.sleep(5)
             except Exception as ex:
-                _LOGGER.error("Listen_event: %s - %s", ex.status, ex.message)
+                _LOGGER.error("Listen_event: %s", ex)
```

Comment:
Are there any known occurrences of other exceptions than ClientResponseError or JSONDecodeError ?

Reply:
I can't recall that I have seen this error myself or seen it mentioned in any reports.